### PR TITLE
Support for hash and modules

### PR DESCRIPTION
**Pull request by yzainee**
1. Added github logic to convert semver to hash, get dates for hash etc
2. Added logic for modules to existing golang scraper logic

### CODE REVIEW

File: f8a_utils/golang_utils.py [hunk 1]
```diff
@@ -84,6 +85,20 @@ def __fetch_gh_link(self, obj):
...
+        module_lst = []
+        if not mod_val:
+            mod_val = obj.get_value('a', {'data-test-id': 'DetailsHeader-infoLabelModule'})
+        if mod_val:
+            module_lst.append(mod_val)
+            if "github" not in mod_val:
+                gh_link = self.get_gh_link()
+                if "https" in gh_link:
+                    module_lst.append(gh_link.split('https://')[1])
+        print(module_lst)
```
Comment (prashbnair) [inline f8a_utils/golang_utils.py:99]: Please remove print statement

File: f8a_utils/gh_utils.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+import random
+import logging
+import requests
+from os import environ
+from datetime import datetime
+
+_logger = logging.getLogger(__name__)
+
+
+class GithubUtils:
```
Comment (prashbnair) [inline f8a_utils/gh_utils.py:29]: where is this going to be used?

File: f8a_utils/golang_utils.py (hunk 1 above)
Comment (yzainee) [reply f8a_utils/golang_utils.py:99]: ack

File: f8a_utils/gh_utils.py (hunk 2 above)
Comment (yzainee) [reply f8a_utils/gh_utils.py:29]: in graph cve sync, api server and backbone

**Review by prashbnair — APPROVED** [review]

**Comment by yzainee** [comment]
[test]

**Pull request merged**